devops/aws-codebuild/Jenkinsfile.cd

```diff
@@ -1,46 +1,45 @@
 #!groovy
 
-def evLibrary = library(identifier: 'evernym-aws-codebuild@v0.4.0-aws-codebuild', retriever: modernSCM(
-    github(credentialsId: 'evernym-github-machine-user', repoOwner: 'evernym', repository: 'jenkins-shared')
-)).com.evernym.pipeline
+def sovLibrary = library(identifier: 'sovrin-aws-codebuild@master', retriever: modernSCM(
+    github(credentialsId: 'sovbot-github', repoOwner: 'sovrin-foundation', repository: 'aws-codebuild-pipeline-plugin')
+)).com.sovrin.pipeline
 
-logger = evLibrary.Logger.new(this)
+logger = sovLibrary.Logger.new(this)
 logger.setGlobalLevel('TRACE')
 
-notifier = evLibrary.Notifier.new(this)
-utils = evLibrary.Utils.new(this)
+notifier = sovLibrary.Notifier.new(this)
+utils = sovLibrary.Utils.new(this)
 
 // TODO set proper labels
 def nodeLabels = [
     codeBuild: env.LIBSOVTOKEN_CODEBUILD_NODE_LABEL ?: 'codebuild',
-    macos: env.LIBSOVTOKEN_MACOS_NODE_LABEL ?: 'macos', 
+    macos: env.LIBSOVTOKEN_MACOS_NODE_LABEL ?: 'macos',
 ]
 
 def codeBuildPipelines = {
     def packageName = 'libsovtoken'
-    
+
     //put code build containers inside a vpc under our dev account
     env.USE_VPC_CONFIG = true
-    env.SOVRIN_REPO_HOST = '192.168.11.115'
+    env.SOVRIN_REPO_HOST = '192.168.101.193'
```

Comment:
We have this stored in a environment variable on the Jenkins instance. It would probably be best to keep IP addresses out of version control and just ask for the ENV variable here.

Reply:
sounds like feature request. please add ticket for feature request.

Reply:
This does not affect security so we will enhance if somebody makes another PR